### DISCUSSION TITLE
How to provide AWS credentials

### DISCUSSION THREAD

**Discussion by Chandrian** [cargo-lambda/cargo-lambda#241] [Q&A] [answered]
Hello, 

Is there a way to provide an aws profile when running the `cargo lambda deploy`? Like the AWS CLI --profile?

Thank you,

**Comment by calavera** [comment, marked as answer]
The same flag should work:

https://github.com/cargo-lambda/cargo-lambda/blob/main/crates/cargo-lambda-remote/src/lib.rs#L13-L15

`cargo lambda deploy --profile`
